src/Promenade.Web/Styles/style.css

```diff
@@ -268,3 +271,13 @@ footer {
 .prom-location .prom-location-nearby td {
     vertical-align: middle;
 }
+.prom-faq-icons {
+    border: 1px solid #b1800e;
+    color: #7f7141;
+}
```

Comment:
This is used in the one of the DB pages for about/help

Reply:
If it's for a DB page it should probably be included in an external resource styling sheet instead of the main one included with the project. I'm not sure what the proper method is for including this in our project.

Reply:
Resolved in commit 475e5f718a25a489084150653e572f1814780056